src/AutoStep/Language/Test/Visitors/FileVisitor.cs

```diff
@@ -68,26 +68,26 @@ public override FileElement VisitFeatureBlock([NotNull] AutoStepParser.FeatureBl
             // Clear the set of feature scenario names.
             featureScenarioNames.Clear();
 
-            if (Result.Feature != null)
+            if (Result!.Feature != null)
```

Comment:
Where you still getting nullability warnings in this file? The Debug.Assert should be taking care of that already....

Reply:
not in release mode

Reply:
Ah, of course. 🤦‍♂️